### PR TITLE
- Golden tests for generator page

### PR DESCRIPTION
**Pull request by suyangdong**

### CODE REVIEW

File: test/golden/generator/generator_page_golden_test.dart [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        }
+      }
+      await multiScreenGolden(
+        tester,
+        'generator_page_devices',
+        devices: [
+          Device.iphone11,
+          Device.tabletLandscape,
+          Device.tabletPortrait
+        ],
```
Comment (ripplearc) [inline test/golden/generator/generator_page_golden_test.dart:109]: It should a standard across all page golden tests

File: lib/main.dart [hunk 2]
```diff
@@ -31,17 +31,26 @@ class MyApp extends StatelessWidget {
...
+            title: "Namer App",
+            theme: ThemeData(
+                useMaterial3: true,
+                colorScheme:
+                    ColorScheme.fromSeed(seedColor: Colors.deepOrange)),
+            home: MyHomePage(
+                generatorPage: const GeneratorPage(),
+                favoritesPage:
+                    FavoritesPage(listKey: GlobalKey<AnimatedListState>()))));
+                // favoritesPage: Container())));
```
Comment (ripplearc) [inline lib/main.dart:43]: remove

File: test/golden/generator/generator_page_golden_test.dart [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  });
+
+  group("Responsive layout of the generator page", () {
+    when(mockRandomWordFactory.generateRandomWord()).thenAnswer((_) {
+      return randomWords.next();
+    });
+    testGoldens("Five history words", (tester) async {
+      final builder = DeviceBuilder()
+        ..overrideDevicesForAllScenarios(devices: DevicePool.devices)
+        // ..overrideDevicesForAllScenarios(devices: [DevicePool.iphone13ProMax])
```
Comment (ripplearc) [inline test/golden/generator/generator_page_golden_test.dart:97]: 👾

**Review by suyangdong — COMMENTED** [review]
LGTM

**Pull request merged**